speedy-recovery-client/src/components/secured/home/HomePageForParent.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+class HomePageForParent extends Component {
+  static defaultProps = {
```

Comment:
what's this?

Reply:
> what's this?
that is use to initialize the properties. because there is a case that we do not have any appointments. so i need to set the time of the appointment to something so i would not get undefined.